frontend/src/components/footer.tsx

```diff
@@ -16,31 +30,86 @@ import OpenStaxURL from '../images/openstax-logo.png'
 
 const Funders = () => {
     return (
-        <Container bg='white'>
-            <Stack>
-                <Title order={4} className="fw-bold">Support from Scientific Agencies</Title>
-                <Flex justify='center' align='center' mx='auto'>
-                    <Anchor target="_blank" href="https://ies.ed.gov/">
-                        <Image alt="Institute of Education Sciences logo" src={IESLogoURL}/>
-                    </Anchor>
+        <Container bg="white" w="100%" p="md">
+            <Stack gap="md">
+                <Title
```

Comment:
@divitcr7 I know the specs in the figma file probably have exact pixel values, however we want everything to appear consistent on the site. We should prefer to just use
`<Title order={X}>` and not customize the line height, font size, font weight, etc... That way other `Title order={X}` will look the same throughout the site. You can remove almost all of these properties I think